psd-web/test/test_helpers/devise.rb

```diff
@@ -1,39 +1,19 @@
 module TestHelpers
-  module OmniAuthHelper
-    def stub_omniauth(user)
-      OmniAuth.config.test_mode = true
-      groups = user.teams.flat_map(&:path) << user.organisation&.path
-      groups.compact!
-      OmniAuth.config.mock_auth[:openid_connect] = OmniAuth::AuthHash.new(
-        "provider" => :openid_connect,
-        "uid"  => user.id,
-        "info" => {
-          "email" => user.email,
-          "name" => user.name,
-        },
-        "extra" => {
-          "raw_info" => {
-            "groups" => groups
-          }
-        }
-      )
-      Rails.application.env_config["omniauth.auth"] = OmniAuth.config.mock_auth[:openid_connect]
-    end
-  end
-
   module Devise
-    include TestHelpers::OmniAuthHelper
-
     def sign_out
       return if page.has_css?("a", text: "Sign in to your account")
 
       click_on "Sign out", match: :first
     end
 
     def sign_in(user = users(:opss))
-      stub_omniauth(user)
       visit root_path
       click_on "Sign in to your account"
+
+      fill_in "user[email]", with: user.email
```

Comment:
I think it's helpful to use the label text in these Capybara helpers, as that additionally tests that the label is correctly associated with the field (via the `for` attribute). Also avoids making any assumption about the `name` value of the field.

```suggestion
      fill_in "Email address", with: user.email
```

Reply:
good point